documenten/techniek/designdecisions.md

```diff
@@ -17,7 +17,7 @@ We hanteren https://docs.geostandaarden.nl/api/API-Strategie/#versioning  als ui
 
 On standards and standardization
 -------
-The specific goal of the proto component (which this current code base is a version of) is to provide a common architecture for common ground components. As such the common ground principles are leading in design choices, and within those principles international compliancy and technological invocation is deemed most important. **We do not want to make concessions to the current infrastructure.** As such the component might differ on [NL API Strategie](https://docs.geostandaarden.nl/api/API-Strategie), [NORA](https://www.noraonline.nl/wiki/Standaarden), [vng.cloud](https://zaakgerichtwerken.vng.cloud/themas/index) and or other standards if they are deemed incompatible or out of line with (inter)national standards and or good practices. 
```

Comment:
Ik heb het stuk over het proto component weggehaald. Het proto component is slechts een framework. Onze keuzes over standaarden zouden los van frameworks moeten zijn

Reply:
True, maar dan zou  ik onderstaande wel laten staan

`The common ground principles are leading in design choices, and within those principles international compliancy and technological invocation is deemed most important. **We do not want to make concessions to the current infrastructure.** As such the components might differ on [NL API Strategie](https://docs.geostandaarden.nl/api/API-Strategie), [NORA](https://www.noraonline.nl/wiki/Standaarden), [vng.cloud](https://zaakgerichtwerken.vng.cloud/themas/index) and or other standards if they are deemed incompatible or out of line with (inter)national standards and or good practices. `

Reply:
Ja dat was ook de bedoeling, staat er ook gewoon nog in mijn versie maar op 1 of andere manier wordt hierboven gesuggereerd dat ik het helemaal heb weggehaald. Kiujk je bij files changed, dan zie je het nog wel staan.

Reply:
Ja daar staat die inderdaad goed :)